test/core/test/fixture-options.test.ts

```diff
@@ -0,0 +1,41 @@
+import { afterAll, beforeEach, describe, expect, test, vi } from 'vitest'
+
+const mockServer = { setup: vi.fn(), teardown: vi.fn() }
+const FnA = vi.fn()
+
+const myTest = test.extend({
+  autoFixture: [async ({}, use) => {
+    await mockServer.setup()
+    await use()
+    await mockServer.teardown()
+  }, { auto: true }],
+
+  normalFixture: [async ({}, use) => {
+    await FnA()
+    await use()
+  }, {}],
```

Comment:
This looks wild, but I suppose this is also what playwright does? (i.e. if there's no `auto` property etc... in 2nd element, then the whole array is considered as a fixture).
https://github.com/microsoft/playwright/blob/6b37a706db4e40d476f8d77b9243898470f5a888/packages/playwright/src/common/fixtures.ts#L65-L67

I think it's totally okay, but I wonder what typing says in this case. is this `normalFixture: void` or it somehow infers array?

Reply:
Yeah it is what Playwright does as far as I know. 

In this case, it should be an array as It's used to demonstrate when the whole array would be considered as a fixture, and users can use `someFixture: void` if they want to use automatic fixtures.